packages/shared/routes/dashboard/staking/views/StakingInfo.svelte

```diff
@@ -1,39 +1,44 @@
 <script lang="typescript">
     import { Animation, Link, Text } from 'shared/components'
     import { Platform } from 'shared/lib/platform'
-    import { localize } from '@core/i18n'
+    import { LocaleArguments, localize } from '@core/i18n'
     import {
         assemblyStakingEventState,
         assemblyStakingRemainingTime,
+        currentAssemblyStakingRewards,
+        currentAssemblyStakingRewardsBelowMinimum,
+        currentShimmerStakingRewards,
+        currentShimmerStakingRewardsBelowMinimum,
         selectedAccountParticipationOverview,
         shimmerStakingEventState,
         stakedAccounts,
     } from 'shared/lib/participation/stores'
     import { ParticipationEventState } from 'shared/lib/participation/types'
     import { getBestTimeDuration } from 'shared/lib/time'
     import { selectedAccountId } from '@lib/wallet'
+    import { Token } from '@lib/typings/assets'
 
     let animation: string
     let header: string
-    let subHeader: string
+    let body: string
 
     let isAssemblyStaked = false
     $: isAssemblyStaked = $selectedAccountParticipationOverview?.assemblyStakedFunds > 0
 
     let isShimmerStaked = false
     $: isShimmerStaked = $selectedAccountParticipationOverview?.shimmerStakedFunds > 0
 
-    $: $selectedAccountParticipationOverview, $assemblyStakingEventState, $shimmerStakingEventState, setHeaders()
+    let stakingEventState = ParticipationEventState.Inactive
+    $: stakingEventState = $assemblyStakingEventState
+
+    $: $selectedAccountParticipationOverview, $assemblyStakingEventState, $shimmerStakingEventState, setText()
     $: isAssemblyStaked,
         isShimmerStaked,
         $assemblyStakingEventState,
         $shimmerStakingEventState,
         $selectedAccountId,
         setAnimation()
```

Comment:
can replace $assemblyStakingEventState, $shimmerStakingEventState with $stakingEventState in these two reactive statements